AKS cluster creation is failing

If I'm reading the [logs](https://devops-ci.elastic.co/job/cloud-on-k8s-e2e-tests-aks/482/console) correctly it looks like a cluster version issue:

```
02:24:47  2020/09/23 00:24:47 Creating cluster...
02:24:48  SSH key files '/root/.ssh/id_rsa' and '/root/.ssh/id_rsa.pub' have been generated under ~/.ssh to allow SSH access to the VM. If using machines without permanent storage like Azure Cloud Shell without an attached file share, back up your keys to a safe location
02:24:53  Operation failed with status: 'Bad Request'. Details: Version 1.16.10 is not supported in this region. Please use [az aks get-versions] command to get the supported version list in this region. For more information, please check https://aka.ms/supported-version-list
02:24:53  Error: exit status 1
02:24:53  exit status 1
02:24:53  make: *** [Makefile:448: run-deployer] Error 1
02:24:53  Makefile:45: recipe for target 'ci-internal' failed
02:24:53  make[1]: *** [ci-internal] Error 2
02:24:53  make[1]: Leaving directory '/var/lib/jenkins/workspace/cloud-on-k8s-e2e-tests-aks/.ci'
02:24:53  Makefile:39: recipe for target 'ci' failed
02:24:53  make: *** [ci] Error 2
```

--- a/hack/deployer/config/plans.yml
+++ b/hack/deployer/config/plans.yml
@@ -47,7 +47,7 @@ plans:
   operation: create
   clusterName: ci
   provider: aks
-  kubernetesVersion: 1.16.10
+  kubernetesVersion: 1.17.11
   machineType: Standard_D8s_v3
   serviceAccount: true
   psp: false
@@ -59,7 +59,7 @@ plans:
   operation: create
   clusterName: dev
   provider: aks
-  kubernetesVersion: 1.16.10
+  kubernetesVersion: 1.17.11
   machineType: Standard_D8s_v3
   serviceAccount: false
   psp: false

